src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -513,8 +513,8 @@ def run(self, timeout):
           'Shared corpus merge finished successfully.',
           output=symbolized_output)
     except engine.TimeoutError as e:
-      logs.log_error('Corpus pruning timed out while merging shared corpus: ' +
-                     repr(e))
+      raise CorpusPruningException(
+          'Corpus pruning timed out while merging shared corpus: ' + repr(e))
```

Comment:
use `\n` at the end of the line, otherwise all these errors will be reported as unique